.github/workflows/check-rtw.yml

```diff
@@ -6,11 +6,15 @@ on:
   # Triggers the workflow on push events
   push:
     paths:
-#      - esmvaltool/utils/recipe_test_workflow/**
+      - esmvaltool/utils/recipe_test_workflow/**
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
+  # Schedule this workflow to run at 04:00 every 10 days
+  schedule:
+    - cron: '0 4 */10 * *'
+
```

Comment:
Added this as suggested from https://github.com/ESMValGroup/ESMValTool/pull/3210#discussion_r1852418018 (thanks @valeriupredoi!) 🥳